source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -490,7 +499,7 @@ struct add_impacts_visitor : public apply_impacts_visitor {
     void operator()(const nt::StopArea* stop_area) {
         this->log_start_action(stop_area->uri);
         for (const auto stop_point : stop_area->stop_point_list) {
-            this->operator()(stop_point);
+            handle_stop_point(stop_point);
```

Comment:
hmmm, this function handles stop_point only for the case where the disruption is applied on stop_area(the behaviour of disruption on stop_area stays the same). However when the disruption is applied on stop_point, nothing happened, since `void operator()(const nt::StopPoint)` is empty. Am I misunderstanding something?